app/js/trouble.js

```diff
@@ -72,38 +78,41 @@
 
 		function open() {
 			$dropdown.removeClass('no_transition');
-			$dropdown.data('status', 'open")'
+			$dropdown.data('status', "open");

```

Comment:
This fixes the linting error, however, you'll want to be consistent about which quotes you use, ' or "